core/src/main/scala/org/genivi/sota/core/data/InstallHistory.scala

```diff
@@ -5,19 +5,34 @@
 package org.genivi.sota.core.data
 
 import org.joda.time.DateTime
+import java.util.UUID
+
+/**
+ * Domain object for the update operation result
+ * @param id The unique id of operation.
+ * @param updateId Id of update this operation belongs to.
+ * @param resultCode The status of operation.
+ * @param resultText The description of operation.
+ */
+case class OperationResult(
+  id : String,
+  updateId: UUID,
```

Comment:
Maybe it's clearer (and easier :)) to have these 'message' case classes in snake_case?